docs/user/library/jdbc/db2.rst

```diff
@@ -69,6 +69,25 @@ For more information check the java docs for:
 * DB2NGJNDIDataStoreFactory
 * DB2NGDataStoreFactory
 
+Advanced Geotools Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++-------------------+------------------------------------------+
+| Parameter         | Description                              |
++===================+==========================================+
+| "loose bbox"      | Flag controlling loose bbox comparisons, |
+|                   | default is true                          |
++-------------------+------------------------------------------+
+| "use selectivity" | Flag to raise the priority of spatial    |
+|                   | indices for the db2 optimizer,           |
+|                   | default is false                         |
++-------------------+------------------------------------------+
```

Comment:
Hi Jody

Feel free to change the wording. I will try to explain in my words.

DB2 uses a cost based optimizer based on distribution statistics. As an example, if you have an index on a  column with  values 'Y' or 'N' and 50% of the records have a value of 'Y' and 50% have a value of 'N', DB2 will ignore the index and will perform a table scan. In this case we have

SELECTIVITY 0.5

This value is calculated from DB2, no need to add it to the select statement.


For geometries (stored as BLOBs) it is not possible to calculate a distribution statistic. The optimizer prefers all other indices before using a spatial index.

Adding SELECTIVITY 0.00001 is a hint to the optimizer. (simulating a distribution statistic). 

Hope this helps
Cheers
Christian